src/fam.rs

```diff
@@ -249,6 +249,22 @@ impl<T: Default + FamStruct> FamStructWrapper<T> {
         Ok(FamStructWrapper { mem_allocator })
     }
 
+    /// Constructs a FamStructWrapper with an empty flexible array member
+    /// from the given FamStruct header.
+    ///
+    /// If the header has a length != 0 set, then we overwrite it with 0.
+    pub fn empty(mut header: T) -> FamStructWrapper<T> {
+        // SAFETY: 0 is always a safe length for a FAM
+        unsafe {
+            header.set_len(0);
```

Comment:
This maybe should be an assert?

Reply:
Yeah, I wasnt quite sure what I wanted here myself - thinking about it now, having it return an error if the length is not zero is probably better API